modules/test-ui/config-node.json

```diff
@@ -25,5 +25,5 @@
     "1337": "http://localhost:8545",
     "1338": "http://localhost:8546"
   },
-  "port": 3333
+  "port": 3000
```

Comment:
should either be `gitignored` or put in the config directory

Reply:
Done.